css/style.css

```diff
@@ -366,7 +493,7 @@ header {
     grid-column: 1 / -1;
     grid-row: 2;
     position: relative;
-    left: -1em;

```

Comment:
Missed comment

Reply:
in teh works. Not sure what the original intent of this comment was supposed to be